Minitwit_BE/Minitwit_BE.Api/Controllers/TwitController.cs

```diff
@@ -21,35 +21,60 @@ public async Task<string> TestEndpoint()
             return await Task.FromResult("test");
         }
 
-        // TODO: Adding to the database should be POST/PUT. The message should be taken from the request body.
-        [HttpGet("add")]
-        public async Task<string> AddTwit()
+        [HttpPost("add")]
+        public async Task<ActionResult<Message>> AddTwit([FromBody]MessageInput input)
```

Comment:
Is it necessary to return the message here? I think it would be sufficient to do `Task<ActionResult>` and then only return Ok() if the message is successful.